tensorboard/plugins/projector/README.md

```diff
@@ -0,0 +1,11 @@
+## Development
+
+To develop the Embedding Projector, launch it in standalone mode:
+```sh
+bazel run tensorboard/plugins/projector/vz_projector:devserver
+```
+
+And open `http://localhost:6006/index.html`. The projector points to the local
```

Comment:
nit/suggestion: make this a link (change `` `url` `` to `<url>`)?

Reply:
Done